.babelrc

```diff
@@ -1,15 +1,34 @@
 {
-  "presets": ["next/babel"], // https://goo.gl/aqbWJY
+  "presets": [[
+    "next/babel",
+    {
+      "preset-env": {
+        "useBuiltIns": "entry",
+        "corejs": 3,
+        "debug": true
+      },
+      "transform-runtime": {
+        "corejs": 3,
+        "useESModules": false
+      },
+      "class-properties": {}
+    }
+  ]], // https://goo.gl/aqbWJY
   "plugins": [
     "transform-flow-strip-types",
-    [
-      "ttag",
-      {
+    "babel-plugin-styled-components"
+  ],
+  "env": {
+    "ttag": {
```

Comment:
This is the configuration used when running Babel with `BABEL_ENV=ttag`.

Reply:
did you test the pot file extraction with these changes?

Reply:
I didn't look in detail at the `.pot` file, but it was generated correctly.